scripts/release.ts

```diff
@@ -66,6 +67,26 @@ async function undoLastCommit() {
   console.log(c.green('Publish commit deleted.'));
 }
 
+const newDesignSystemVersion = (): string => {
+  const newDesignSystemVersionJSON = JSON.parse(sh(`npm version -w @cmsgov/design-system --json`));
+  return newDesignSystemVersionJSON['@cmsgov/design-system'];
+};
+
+const updateDSVersion = (json: JSON | any): JSON => {
+  const dsKey = '@cmsgov/design-system';
+  json.dependencies[dsKey] = newDesignSystemVersion();
+  return json;
```

Comment:
Would it be possible to add a line break at the end of this json?

The file diff reads `\ No newline at end of file` for each of the package.json files that were updated after running `yarn release`

Reply:
I think that might have to do with the fact that I'm then requiring in the JSON rather than reading the file directly. I'll make that change and we'll if it fixes it!